Design again

aviama opened the pull request:
lint siger brug button, men det fungerer ik, så jeg har brugt en input button

bruger ogs duplicate id, det skal fikses engang

aviama committed:
kinda fixed the settings bar thing

aviama committed:
pulled out the playbar from the controls pane

aviama committed:
new vertalign button

aviama committed:
mini fix. unsued const decl

aviama committed:
stuff

aviama committed:
Merge branch 'main' into designAgain

atjn commented on source/style.css:
```diff
@@ -209,41 +206,29 @@ footer {
+	z-index: 200;
+}
+
+h3#controls-pane {
```
```suggestion
#controls-pane h3 {
```
As far as I can tell, we don't have an `h3` with id `controls-pane` but we do have a `controls-pane` with an `h3` inside :)
Maybe the plan was to delete these styles?

atjn replied:
I fixed this for you in #27

atjn committed:
Ensure settings button placement

Before this change, if you changed the height of the `<h3>`, you would also move the settings button.

atjn committed:
Fix lint issues

atjn committed:
Fix dual IDs

atjn committed:
Streamline button designs

atjn committed:
Fix CSS specificity issue

atjn commented:
I've fixed the button styling issues now. Feel free to change the design I used.
